addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/receipt_header/receipt_header.xml

```diff
@@ -22,7 +22,7 @@
                     <div t-esc="props.data.cashier" />
                 </div>
                 <div t-if="props.data.trackingNumber and !props.data.bigTrackingNumber">
-                    <span class="fs-2" t-esc="props.data.trackingNumber" />
+                    <span class="tracking-number" style="font-size: 2rem;" t-esc="props.data.trackingNumber" />
```

Comment:
why do you need to add the class ?

Reply:
Sorry i forgot to mention related pr in the comit message. I have changed it because it was used for x-path but now removed it and used fs-1 for the xpath.